### PR TITLE
Refactor/branding docs

### PR DESCRIPTION
**Pull request by petrKavulok**
in this PR

- Branding docs have been updated to serve as a guide for graphic designers
- Padding assigned to minimized versions of branding images in sidebar
- card header title height issue was fixed

### CODE REVIEW

File: doc/branding.md [hunk 1]
```diff
@@ -72,15 +72,30 @@ With the configuration set as above, the title will be: `TeskaLabs | Page Title`
 
 Every image HAS TO be provided in SVG (vectorized).
 Use of pixel formats (PNG, JPG, ...) is strongly discouraged.
+While creating the branding images, use full width/height of the canvas (ratio 3:1 on full and 1:1 on minimized version). No padding is required for optimal viewing experience.
 
 ### Branding images
 
-Top-left location.
+format: SVG
```
Comment (Pe5h4) [inline doc/branding.md:79]: @petrKavulok Not sure if it is written by markdown standards. What I mean is that for new lines you need to have an empty line between the text. For indentations its better to use some bullet list. Pls update accordingly and pls take a look how it looks like when rendered as markdown (I bet visualstudio has some package for it)

File: doc/branding.md [hunk 2]
```diff
@@ -72,15 +72,30 @@ With the configuration set as above, the title will be: `TeskaLabs | Page Title`
...
+
+Logo should be suitable for use in both light & dark mode. 
 
 Two sizes:
 
  * 120x30 (full)
  * 30x30 (minimized)
 
 
+SidebarLogo
```
Comment (Pe5h4) [inline doc/branding.md:97]: @petrKavulok Here the same, it does not comply with markdown standards

**Review by Pe5h4 — CHANGES_REQUESTED** [review]

File: doc/branding.md [hunk 3]
```diff
@@ -72,15 +72,30 @@ With the configuration set as above, the title will be: `TeskaLabs | Page Title`
 
 Every image HAS TO be provided in SVG (vectorized).
 Use of pixel formats (PNG, JPG, ...) is strongly discouraged.
+While creating the branding images, use full width/height of the canvas (ratio 3:1 on full and 1:1 on minimized version). <em>No padding is required for optimal viewing experience.</em>
 
 ### Branding images
 
-Top-left location.
+format: **SVG**
```
Comment (Pe5h4) [inline doc/branding.md:79]: @petrKavulok Not sure if you took a look how does it look like in some markdown compiler, but this is still not correct. If you want to have it on separate line, you need to add an empty newline below the text :)

Comment (petrKavulok) [reply doc/branding.md:79]: > @petrKavulok Not sure if you took a look how does it look like in some markdown compiler, but this is still not correct. If you want to have it on separate line, you need to add an empty newline below the text :)

I did check it with this 'markdown preview' extension. 
I don't think I understand what you mean 

Snímek obrazovky 2022-08-04 v 12 54 14

Comment (Pe5h4) [reply doc/branding.md:79]: @petrKavulok Just take a look here in github preview: https://github.com/TeskaLabs/asab-webui/blob/e62719cc3fbaf4487162513bcd9a44be216feb67/doc/branding.md

if you want to have a newline in markdown, you need to add an empty newline between the rows

```
format: **SVG**

full:
...
```
It seems like a markdown compiler of visualstudio "schroustne vsechno" :)

**Review by Pe5h4 — APPROVED** [review]
@petrKavulok Pls fix what I wrote here https://github.com/TeskaLabs/asab-webui/pull/341/files#r937986040
solve merge conflicts and you are free to merge it

**Pull request merged**